modules/remotecontrol/Controller.js

```diff
@@ -42,41 +44,56 @@ function getModifiers(event) {
  * It listens for mouse and keyboard events and sends them to the receiver
  * party of the remote control session.
  */
-class Controller {
+export default class Controller {
     /**
      * Creates new instance.
      */
-    constructor() {}
+    constructor() {
+        this.enabled = false;
+    }
+
+    /**
+     * Enables / Disables the remote control
+     * @param {boolean} enabled the new state.
+     */
+    enable(enabled) {
+        this.enabled = enabled;
+    }
 
     /**
      * Starts processing the mouse and keyboard events.
      * @param {JQuery.selector} area the selector which will be used for
      * attaching the listeners on.
      */
     start(area) {
+        if(!this.enabled)
```

Comment:
Is it expected that Controller will be started if not enabled ? I would rather see here some error logged or thrown in such case instead of ignoring it silently.